datasets/iowa_liquor_sales/docs/tutorials/sales_prediction/sales_prediction_test.py

```diff
@@ -0,0 +1,33 @@
+import pytest
+from testbook import testbook
+
+
+@pytest.mark.timeout(900)
+@testbook(
+    "datasets/iowa_liquor_sales/docs/tutorials/liquor_sales_prediction/liquor_sales_predictions.ipynb"
+)
+def test_run_notebook(tb):
+    tb.inject(
+        """
+        from unittest import mock
+        import pandas as pd
+        mock_client = mock.MagicMock()
+        mock_df = pd.DataFrame()
+        mock_df['week'] = range(50)
+        mock_df['count'] = range(50)
+        mock_df['category'] = 'Tree Maintenance'
+        p1 = mock.patch.object(bigquery, 'Client', return_value=mock_client)
+        mock_client.query().result().to_dataframe.return_value = mock_df
+        p1.start()
+        """,
+        before=3,
+        run=False,
+    )
+
+    tb.execute()
+    dataframe = tb.get("dataframe")
+    assert dataframe.shape == (50, 3)
+
+    train_pred_plot = tb.get("train_pred_plot")
```

Comment:
You want to pick an object from your code (ideally something from the end of your code) and replace `train_pred_plot ` with it, and test whether it exists or not. That way, we are testing whether your notebook runs all the way to the end during the test.